README-FRONTEND.md

```diff
@@ -12,7 +12,7 @@ For this challenge we would like you to:
 
 - Populate items in the left-hand sidebar with data from the /api/items endpoint.
 - Allow users to select items in the left-hand sidebar and have them appear in the Menu Preview on the right hand side.
-- Allow users to filter items in the left-hand sidebar by name.
+- Allow users to filter items in the left-hand sidebar by name. We prefer this to be done on the server.
```

Comment:
I would change this to [TIP: you can modify the server code]

Reply:
```suggestion
- Allow users to filter items in the left-hand sidebar by name. You can modify the server code.
```